simulationworkflowschema/molecular_dynamics.py

```diff
@@ -1039,6 +1294,10 @@ class MolecularDynamicsResults(ThermodynamicsResults):
         sub_section=MeanSquaredDisplacement, repeats=True
     )
 
+    free_energy_calculations = SubSection(
+        sub_section=FreeEnergyCalculations, repeats=True
```

Comment:
Is this missing a `m_def`? @ladinesa ?

Reply:
I just added m_def to all of the sections in this class